platform/ui/src/components/modal/Modal.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Modal from 'react-bootstrap-modal';
+
+const CustomModal = ({
+  className,
+  closeButton,
+  backdrop,
+  keyboard,
+  show,
+  large,
+  title,
+  onHide,
+  footer: Footer,
+  header: Header,
+  children: Component,
+}) => (
+  <Modal
+    className={`modal fade themed in ${className}`}
```

Comment:
can we use our `classnames` dependency here to clean this up instead of using string interpolation?

Reply:
Done! thx.